packages/front-end/components/Experiment/StartExperimentBanner.tsx

```diff
@@ -258,8 +377,23 @@ export function StartExperimentBanner({
           <Tooltip body={"We're not able to verify these automatically"} />
           <ul style={{ fontSize: "1.1em" }} className="ml-0 pl-0 mb-0 pb-0">
             {manualChecklist.map((item, i) => (
-              <li key={i} style={{ listStyleType: "none", marginLeft: 0 }}>
-                &bull; {item}
+              <li
+                key={i}
+                style={{
+                  listStyleType: "none",
+                  marginLeft: 0,
+                  marginBottom: 3,
+                }}
+              >
+                <input
+                  type="checkbox"
+                  className="ml-0 pl-0"
+                  checked={isTaskCompleted(item.key)}
+                  onChange={async (e) =>
+                    updateTaskStatus(e.target.checked, item)
```

Comment:
This design is vulnerable to two types of race conditions:
1. If you double click a checklist item quickly, it will fire multiple API calls with conflicting states and it's not guaranteed which order they will be processed in.
2. If you click a 2nd checklist item before the API call finishes for the 1st one, it will overwrite the first change

The entire manual checklist area needs a "updating" state that disables inputs while API calls are in flight.  You can see this on GitHub in action - add a TODO list to a PR comment with multiple items and click one and you'll see the whole form gray out for a second while it's recording your change.